server/impl/context.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/gorilla/sessions"
 )
 
+//go:generate go run golang.org/x/tools/cmd/stringer -type=contextKey
```

Comment:
[stringer](https://pkg.go.dev/golang.org/x/tools/cmd/stringer) automatically generates a .String() method for contextKey based on the constant names.